### DISCUSSION TITLE
Should we implement a separate comment API from fb?

### DISCUSSION THREAD

**Discussion by tachib-akiko** [anime-vsub/desktop-web#44] [Q&A]
Hey @tachibana-shin. the original site is currently using Facebook comment plugin. Should we create an API to send to Facebook to implement our own comments?

**Comment by tachibana-shin** [comment]
> Hey @tachibana-shin. the original site is currently using Facebook comment plugin. Should we create an API to send to Facebook to implement our own comments?

I think it's complicated. somehow with 3rd party cookie policy we will still have to embed fb comments because we can't create an api something like sending comments. that needs to get the user's token

**Reply by tachib-akiko** [reply]
that's right. Tired of facebook. they don't even let us customize any css in their plugin
